resilience4j-framework-common/src/main/java/io/github/resilience4j/common/ratelimiter/configuration/RateLimiterConfigurationProperties.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright 2019 Dan Maas
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.resilience4j.common.ratelimiter.configuration;
+
+import io.github.resilience4j.core.ConfigurationNotFoundException;
+import io.github.resilience4j.core.StringUtils;
+import io.github.resilience4j.core.lang.Nullable;
+import io.github.resilience4j.ratelimiter.RateLimiterConfig;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RateLimiterConfigurationProperties {
+
+	private Map<String, BackendProperties> backends = new HashMap<>();
```

Comment:
Unfortunately this breaks backward compatibility.
The rateLimiter used unfortunately `limiters` instead of `backends`.
I really don't like the name `backends` anyway. Maybe we should rename it to `instances`.
For backward compatibility, we could provide both options. 
What do you think?

Reply:
@RobWin yeah - instances works. We can provide both for limiters. Do we need only `instances` for the others?

Reply:
I guess we just need a getter like getBackends() which returns the instances Map